.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,14 @@
 <!-- Provide a general summary of your changes in the Title above -->
 
-## Description
+
+## Description of [new feature/bug fix/ documentation update]
+<!-- If this change breaks (or has the potential to break) existing functionality, add the `breaking` label. -->
```

Comment:
Not all PR authors have privileges to add a label.

Reply:
Should we say:

> Please make sure to mention whether this change breaks (or has the potential to break) existing functionality.

Whoever triages this, should add the `breaking` label then.

Reply:
Agreed